<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post2dWindowParticlesBaseScalarGroupDataItem_ToolBarWidget</class>
 <widget class="QWidget" name="Post2dWindowParticlesBaseScalarGroupDataItem_ToolBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorEditWidget" name="colorWidget">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>Point Size:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="pointSizeSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TransparencyWidget" name="transparencyWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
